apps/studio/components/interfaces/Database/Publications/PublicationsTables.tsx

```diff
@@ -95,38 +87,32 @@ export const PublicationsTables = ({
           <NoSearchResults />
         ) : (
           <div>
-            <Table
-              head={[
-                <Table.th key="header-name">Name</Table.th>,
-                <Table.th key="header-schema">Schema</Table.th>,
-                <Table.th key="header-desc" className="hidden text-left lg:table-cell">
-                  Description
-                </Table.th>,
-                <Table.th key="header-all">
-                  {/* Temporarily disable All tables toggle for publications. See https://github.com/supabase/supabase/pull/7233.
-              <div className="flex flex-row space-x-3 items-center justify-end">
-                <div className="text-xs leading-4 font-medium text-gray-400 text-right ">
-                  All Tables
-                </div>
-                <Toggle
-                  size="tiny"
-                  align="right"
-                  error=""
-                  className="m-0 p-0 ml-2 mt-1 -mb-1"
-                  checked={enabledForAllTables}
-                  onChange={() => toggleReplicationForAllTables(publication, enabledForAllTables)}
-                />
-              </div> */}
-                </Table.th>,
-              ]}
-              body={tables.map((table) => (
-                <PublicationsTableItem
-                  key={table.id}
-                  table={table}
-                  selectedPublication={selectedPublication}
-                />
-              ))}
-            />
+            <Card>
+              <Table>
+                <TableHeader>
+                  <TableRow>
+                    <TableHead>Name</TableHead>
+                    <TableHead>Schema</TableHead>
+                    <TableHead>Description</TableHead>
+                    {/* 
+                      We've disabled All tables toggle for publications. 
+                      See https://github.com/supabase/supabase/pull/7233. 
+                    */}
+                    <TableHead />
+                  </TableRow>
+                </TableHeader>
+                <TableBody>
+                  {!!selectedPublication &&
```

Comment:
This is the current state for invalid publication ID, should we have a more descriptive UI message?

<img width="1182" height="592" alt="Image" src="https://github.com/user-attachments/assets/790aa5fe-2126-41dd-a65d-48597746988c" />

Reply:
oh yes great catch!